apis/config/v1beta1/configuration_types.go

```diff
@@ -184,23 +184,23 @@ type ControllerConfigurationSpec struct {
 	CacheSyncTimeout *time.Duration `json:"cacheSyncTimeout,omitempty"`
 }
 
+// WaitForPodsReady defines configuration for the Wait For Pods Ready feature,
+// which is used to ensure that all Pods are ready within the specified time.
 type WaitForPodsReady struct {
-	// Enable when true, indicates that each admitted workload
-	// blocks the admission of all other workloads from all queues until it is in the
-	// `PodsReady` condition. If false, all workloads start as soon as they are
-	// admitted and do not block admission of other workloads. The PodsReady
-	// condition is only added if this setting is enabled. It defaults to false.
+	// Enable indicates whether to enable wait for pods ready feature.
+	// Defaults to false.
 	Enable bool `json:"enable,omitempty"`
 
 	// Timeout defines the time for an admitted workload to reach the
-	// PodsReady=true condition. When the timeout is reached, the workload admission
-	// is cancelled and requeued in the same cluster queue. Defaults to 5min.
+	// PodsReady=true condition. When the timeout is exceeded, the workload
+	// evicted and requeued in the same cluster queue.
+	// Defaults to 5min.
 	// +optional
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
 
-	// BlockAdmission when true, cluster queue will block admissions for all subsequent jobs
-	// until the jobs reach the PodsReady=true condition. It defaults to false if Enable is false
-	// and defaults to true otherwise.
+	// BlockAdmission when true, cluster queue will block admissions for all
+	// subsequent jobs until the jobs reach the PodsReady=true condition.
+	// It defaults to false if Enable is false and defaults to true otherwise.
```

Comment:
uhm...
Maybe we can say that this setting is only honored if `Enable` is set to true, instead of playing words about what the default is.

Reply:
Adjusted. I hope we could also make the defaulting logic simpler in the future. maybe just set it to false regardless or Enable?